src/js/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -0,0 +1,624 @@
+  'Israel (Tel Aviv)',
+  'Italy',
+  'Jamaica',
+  'Japan',
+  'Jordan',
+  'Kazakhstan',
+  'Kenya',
+  'Kosovo',
+  'Kuwait',
+  'Kyrgyzstan',
+  'Laos',
+  'Latvia',
+  'Lebanon',
+  'Leeward Islands',
+  'Lesotho',
+  'Liberia',
+  'Libya',
+  'Liechtenstein',
+  'Lithuania',
+  'Luxembourg',
+  'Macao',
+  'Macedonia',
+  'Madagascar',
+  'Malawi',
+  'Malaysia',
+  'Mali',
+  'Malta',
+  'Martinique',
+  'Mauritania',
+  'Mauritius',
+  'Mexico',
+  'Moldavia',
+  'Mongolia',
+  'Montenegro',
+  'Montserrat',
+  'Morocco',
+  'Mozambique',
+  'Namibia',
+  'Nepal',
+  'Netherlands',
+  'Netherlands Antilles',
+  'Nevis',
+  'New Caledonia',
+  'New Zealand',
+  'Nicaragua',
+  'Niger',
+  'Nigeria',
+  'North Korea',
+  'Northern Ireland',
+  'Norway',
+  'Oman',
+  'Pakistan',
+  'Panama',
+  'Papua New Guinea',
+  'Paraguay',
+  'Peru',
+  'Philippines',
+  'Philippines (restricted payments)',
+  'Poland',
+  'Portugal',
+  'Qatar',
+  'Republic of Yemen',
+  'Romania',
+  'Russia',
+  'Rwanda',
+  'Sao-Tome/Principe',
+  'Saudi Arabia',
+  'Scotland',
+  'Senegal',
+  'Serbia',
+  'Serbia/Montenegro',
+  'Seychelles',
+  'Sicily',
+  'Sierra Leone',
+  'Singapore',
+  'Slovakia',
+  'Slovenia',
+  'Somalia',
+  'South Africa',
+  'South Korea',
+  'Spain',
+  'Sri Lanka',
+  'St. Kitts',
+  'St. Lucia',
+  'St. Vincent',
+  'Sudan',
+  'Suriname',
+  'Swaziland',
+  'Sweden',
+  'Switzerland',
+  'Syria',
+  'Taiwan',
+  'Tajikistan',
+  'Tanzania',
+  'Thailand',
+  'Togo',
+  'Trinidad and Tobago',
+  'Tunisia',
+  'Turkey (Adana only)',
+  'Turkey (except Adana)',
+  'Turkmenistan',
+  'Uganda',
+  'Ukraine',
+  'United Arab Emirates',
+  'United Kingdom',
+  'Uruguay',
+  'Uzbekistan',
+  'Vanuatu',
+  'Venezuela',
+  'Vietnam',
+  'Wales',
+  'Western Samoa',
+  'Yemen Arab Republic',
+  'Zambia',
+  'Zimbabwe'
+];
+
+export const militaryPostOfficeTypeLabels = { // TODO: determine whether these are necessary
+  APO: 'Army Post Office',
+  FPO: 'Fleet Post Office',
+  DPO: 'Diplomatic Post Office'
+};
+
+const addressUISchema = (addressName, title) => {
+  return {
+    'ui:title': title,
+    type: {
+      'ui:title': 'Type',
+      'ui:options': {
+        labels: typeLabels
+      }
+    },
+    country: {
+      'ui:title': 'Country'
+    },
+    state: {
+      'ui:title': 'State',
+      'ui:options': {
+        labels: stateLabels,
+        hideIf: formData => {
+          return (
+            _.get(formData, `veteran[${addressName}].country`) !== 'USA'
+          );
+        }
+      }
+    },
+    addressLine1: {
+      'ui:title': 'Street'
+    },
+    addressLine2: {
+      'ui:title': 'Line 2'
+    },
+    addressLine3: {
+      'ui:title': 'Line 3'
+    },
+    city: {
+      'ui:title': 'City'
+    },
+    militaryStateCode: {
+      'ui:title': 'Military State Code',
+      'ui:options': {
+        labels: stateLabels,
+        hideIf: formData => _.get(formData, `veteran[${addressName}].type`) !== 'MILITARY'
```

Comment:
@markgreenburg you mentioned that this is an implementation detail rather than an option, should the military address fields always be displayed?

Reply:
I'll change this to "United States" instead as I see that used elsewhere.